esp/src/src-react/components/DFUWorkunitDetails.tsx

```diff
@@ -74,6 +77,21 @@ export const DFUWorkunitDetails: React.FunctionComponent<DFUWorkunitDetailsProps
         !workunit?.isDeleted()
     );
 
+    const saveWorkunit = React.useCallback(() => {
+        Promise.resolve(workunit?.update({ JobName: jobname, isProtected: _protected }))
```

Comment:
No need for a Promise.resolve here

Reply:
It was needed because `workunit` here is coming from the older ESPDFUWorkunit, and that update doesn't return a promise-like, to have the subsequent things happen on success, or else catch/log an error. A newer hook using comms for DFU workunits hasn't been made yet. I actually started adding one before opening this, but it was looking like a fairly large change for the sake of the save success message of this PR.

Reply:
Ahhh - ok that makes sense!